libvast/src/format/pcap.cpp

```diff
@@ -410,6 +416,7 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
 reader::flow_state& reader::state(const flow& x) {
   auto i = flows_.find(x);
   if (i == flows_.end()) {
+    // Why do we construct a separate flow here as opposed to using x? --MV
     auto cf = flow{x.src_addr, x.dst_addr, x.src_port, x.dst_port};
```

Comment:
As far as I can tell there's no difference really. We shouldn't copy. Can you make this change in a separate commit?